notification-portlet-webcomponents/notification-banner/src/components/NotificationBanner.vue

```diff
@@ -1,19 +1,28 @@
 <template>
   <div class="notification-banner">
-    <b-alert varient="info" show>
-      Hello World!
+    <b-alert v-for="(notification, index) in notifications" :key="index" varient="info" show>
+      <h3>
+        <i class="fas fa-info"></i>
+        {{notification.title}}
+      </h3>
+      <span v-html="notification.body" />
     </b-alert>
   </div>
 </template>
 
 <script>
 import bAlert from "bootstrap-vue/es/components/alert/alert";
 import oidc from "@uportal/open-id-connect/esm/open-id-connect";
+import { get } from "axios";
```

Comment:
I don't remember installing anything called `axios`, but it seems to be working.

Reply:
I'd install `axios` here just to be safe